dojoWordle/components/Game/Game.style.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from "@emotion/native";
+
+export const ResultContainer = styled.View({
+  alignItems: "center",
+  flexDirection: "row",
+  justifyContent: "center",
+  paddingBottom: 50,
```

Comment:
use spacer if you want to make space between 2 components

Reply:
Why ? 
If you want to re-use ResultContair, it is weird to have paddingBottom: 50 on it. Maybe, in another use case, you will nee to have paddingTop